engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/layout/pdf/util/HTML2Content.java

```diff
@@ -829,10 +831,10 @@ private static IForeignContent getForeignRoot(IContent content) {
 
 	/**
 	 * Outputs the image
-	 * 
+	 *
 	 * @param ele the IMG element instance
 	 */
-	protected static void outputImg(Element ele, Map cssStyles, IContent content) {
+	protected static void outputImg(Element ele, Map<Element, StyleProperties> cssStyles, IContent content) {
```

Comment:
The retrofitting of generics are not supposed to break API or ABI, that was one of the design goals if I remember correctly.
Actually all places that called this method already had generic information, that is why only one line is changed. 
I just wanted to get rid of the some warnings. :-)
Should I remove this change? It do add some unnecessary "noise" to the PR and really belong to #706 instead.

Reply:
> Actually all places that called this method already had generic information,

Since this method is API, we can't know the usage. Also, some people mark warnings as errors in their build. For example @merks. Ed, can you advice us here?